src/gpa/activateGPA.js

```diff
@@ -2,102 +2,95 @@
 Source: https://github.com/IITH/aims-gpa-calculator
 */
 
-var exclude_list = [
-  'Minor core',
-  'Honors core',
-  'Honours project',
-  'Honours coursework',
-  'FCC',
-  'Additional'
-];
+var exclude_list = ['Minor core', 'Honors core', 'Honours project', 'Honours coursework', 'FCC'];
 var grade_values = {
-  'A+': 10,
-  'A': 10,
-  'A-': 9,
-  'B': 8,
-  'B-': 7,
-  'C': 6,
-  'C-': 5,
-  'D': 4,
-  'FR': 0,
-  'FS': 0
+	'A+': 10,
+	'A': 10,
+	'A-': 9,
+	'B': 8,
+	'B-': 7,
+	'C': 6,
+	'C-': 5,
+	'D': 4,
+	'FR': 0,
+	'FS': 0,
+	'S': 0
```

Comment:
Also please explain why `'S'` has to be a key in the JSON Object.

Reply:
I feel that setting S to zero is misleading too. What's your take on this @dark-trojan, @AdaephonBen?